src/styles/index.scss

```diff
@@ -1,3 +1,277 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
 }
+
+.catalog {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+}
+
+/* #region-Header */
+$linkColor: #060b35;
+$activ: #00acdc;
+
+.nav {
+  display: flex;
+  background: #fff;
+  height: 60px;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+  gap: 20px;
+}
+
+.nav :last-child {
```

Comment:
The selector `.nav :last-child` might not work as expected due to its use of a descendant combinator. If you intend to target the last direct child of `.nav`, consider using `.nav > :last-child`.